ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/TransactionTransmitter.java

```diff
@@ -100,12 +108,21 @@ private void sendTransaction(final Buffer bodyContent) {
         ethNodeClient.request(
             httpServerRequest.method(), httpServerRequest.uri(), this::handleResponse);
 
+    request.setTimeout(httpRequestTimeout.toMillis());
+    request.exceptionHandler(thrown -> exceptionHandler(context.getRoutingContext(), thrown));
```

Comment:
we don't need the full context, maybe just pass in fail method?

Reply:
See passthrough - this allows us to do more precise failures.